pycytominer/operations/transform.py

```diff
@@ -13,14 +13,14 @@
 
 
 class Spherize(BaseEstimator, TransformerMixin):
-    """Class to apply a sphering transform (aka whitening) data in the base sklearn
-    transform API. Note, this implementation is modified/inspired from the following
-    sources:
+    """Class to apply a sphering transform (aka whitening) data in the base sklearn transform API.
+
+    This implementation is modified/inspired from the following sources:
     1) A custom function written by Juan C. Caicedo
     2) A custom ZCA function at https://github.com/mwv/zca
     3) Notes from Niranj Chandrasekaran (https://github.com/cytomining/pycytominer/issues/90)
     4) The R package "whitening" written by Strimmer et al (http://strimmerlab.org/software/whitening/)
-    5) Kessy et al. 2016 "Optimal Whitening and Decorrelation" [1]_
+    5) Kessy et al. 2016 "Optimal Whitening and Decorrelation" [1]_.
```

Comment:
It looks like `[1]_` doesn't properly link to the reference for this module's docstring. Consider updating this so the link works properly or moving to a simpler form (`please see the references for this module`, etc.).